### DISCUSSION TITLE
Why only add <style></style> in browser (Remix)

### DISCUSSION THREAD

**Discussion by HurricanKai** [master-co/css#11] [Question]
The remix docs mention this line `{typeof window !== 'undefined' && window && <style></style>}`, if my understand is correct this is suggesting to only include the empty style element (I guess that master styles populates) on the client side. Why is this the case? Seems like including it on the server side has no downsides?

In general I'm unsure how SSR with this can work. Seems like it's only intended for client side usage. Is that because it breaks on hydration (-> using it only on the server is an option) or does it need some client side information?

**Comment by 0Lola** [comment]
Yes, this issue will fixed at https://github.com/master-co/styles/issues/10

Thanks for your reminder. 🥳
